pkg/compose/up.go

```diff
@@ -119,10 +119,14 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, options
 					gracefulTeardown()
 				} else {
 					eg.Go(func() error {
-						return s.Kill(context.Background(), project.Name, api.KillOptions{
+						// Intentionally ignore errors, for cases where some
+						// of the containers are already stopped.
+						s.kill(context.Background(), project.Name, api.KillOptions{
```

Comment:
this should not ignore _all_ errors, but only the "not found" one

Reply:
I don't find a reliable way to distinguish errors here, as they don't seem to be typed. Should I add the check based on the text of the error? I guess it would be to check if the error message contains `is not running`.

Also, with the current code, even if it returns the error, it doesn't seem to be doing anything about it. If the kill fails, the loop is still finished and the command hangs if there are still containers running. I could try to fix this too if I add error handling.

Reply:
have you tried using `errdefs.IsNotFound(err)` ?

Reply:
Thanks! This worked. I also checked `errdefs.IsConflict`, which happens if one of the containers is already stopped.